highs/lp_data/HighsInterface.cpp

```diff
@@ -321,6 +321,9 @@ HighsStatus Highs::formStandardFormLp() {
     this->standard_form_matrix_.start_.push_back(
         HighsInt(this->standard_form_matrix_.index_.size()));
   }
+  // Now set correct values for the dimensions of this->standard_form_matrix_
```

Comment:
I only inserted these three lines. I guess the other changes are down to clang-format